src/com/rath/rathbot/cmd/disc/actions/KickCmd.java

```diff
@@ -1,6 +1,104 @@
+
 package com.rath.rathbot.cmd.disc.actions;
 
+import com.rath.rathbot.RathBot;
+import com.rath.rathbot.cmd.PermissionsTable;
+import com.rath.rathbot.cmd.RBCommand;
+import com.rath.rathbot.msg.MessageHelper;
+
+import sx.blah.discord.api.IDiscordClient;
+import sx.blah.discord.handle.obj.IMessage;
+import sx.blah.discord.handle.obj.IUser;
+
+/**
+ * Kicks user by UID or @mention for a given reason.
+ * 
+ * @author Kami lehenbnw@gmail.com
```

Comment:
Be consistent with your username/email. Also add yourself as an author to RathBot.java because you edited it.

Reply:
Fucking fixed, I swear I did this when I was making that change...